stock_reserve_area/models/stock_reserve_area.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2023 ForgeFlow SL.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class StockReserveArea(models.Model):
+    _name = "stock.reserve.area"
```

Comment:
I miss an action to see current reservations information. It can be generic and the be also accesible from different places: for instance: the stock moves (operations tab in picking)?, maybe a smart button in the picking?